disasm/isa_parser.cc

```diff
@@ -385,6 +387,9 @@ isa_parser_t::isa_parser_t(const char* str, const char *priv)
   if (extension_table[EXT_ZABHA] && !extension_table['A'] && !extension_table[EXT_ZAAMO]) {
     bad_isa_string(str, "'Zabha' extension requires either the 'A' or the 'Zaamo' extension");
   }
+  if (extension_table[EXT_ZAWRS] && !extension_table['A'] && !extension_table[EXT_ZALRSC]) {
+    bad_isa_string(str, "'Zabha' extension requires either the 'A' or the 'Zalrsc' extension");
```

Comment:
We can delete the dependence on `A` here, since at this point, because of https://github.com/riscv-software-src/riscv-isa-sim/blob/e2834ca9e7ae1a3f97cf2f68f185c604831285ae/disasm/isa_parser.cc#L341

Reply:
And we can do the same thing for `Zabha` just above, for the same reason

Reply:
@aswaterman updated. Applied same reason to `Zacas` as well.